test/unit/select_plan_test.lua

```diff
@@ -256,7 +256,7 @@ g.test_first = function()
     t.assert_equals(plan.scan_value, {777}) -- after_tuple id
     t.assert_equals(plan.after_tuple, after_tuple)
     t.assert_equals(plan.scan_condition_num, nil)
-    t.assert_equals(plan.tarantool_iter, box.index.LE) -- inverted iterator
+    t.assert_equals(plan.tarantool_iter, box.index.LT) -- inverted iterator
```

Comment:
Maybe it worth to add more unit test that shows that EQ iterator is changed to GT if after tuple is passed?